apps/site/pages/docs/getting-started/getting-started-overview.mdx

```diff
@@ -17,18 +17,25 @@ This documentation is currently under development.
 
 In this Getting Started track, you'll build and deploy a FastStore Storefront at the domain `https://<account-name>.vtex.app/`.
 
-By the end of this track, you'll have a fully functional store using FastStore, seamlessly integrated with VTEX native solutions like [Headless CMS](/tbd), [Catalog](/tbd), and [Checkout](/tbd).
+By the end of this track, you'll have a fully functional store using FastStore, seamlessly integrated with VTEX native solutions like [Headless CMS]([/tbd](https://www.faststore.dev/tutorials/cms/0)).
 Here's an example of the final result: [Starter Store](https://starter.vtex.app).
-
-**1. [FastStore Onboarding](/tbd)** 
+  
+## 1. [FastStore Onboarding](/docs/getting-started/onboarding)
 
 Start your project with the FastStore Onboarding app. It streamlines the process by setting up the essential store configurations, creating a Github repository for your store, and providing a live demo preview of your store.
 
-**2. [Setting Up the Project](/tbd)**
+## 2. [Setting Up the Project](/docs/getting-started/settingup-project)
+
+Discover the core technologies we will use in this tutorial and set up all the necessary tools to implement your very first FastStore storefront and create your first storefront customization.
```

Comment:
```suggestion
Discover the core technologies that we will use in this tutorial and set up all the necessary tools to implement your very first FastStore storefront and create your first storefront customization.
```

Reply:
Thanks for the suggestion, Guila. I made a slight change to the sentence to have a clear message.